Update 02.introduction.md (#562)
greenelab/covid19-review: content/02.introduction.md

The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
The possibility of asymptomatic cases is also being investigated.
The transmission and mortality rate estimations of COVID-19 remain to be determined.
<!-- I think there are better estimates now (about 50% asymptomatic I think, once you take out the presymptomatic).-->
<!-- Most credible IFR estimates I've seen are between 0.5-1.5%, depending on the population. CFRs will be greater and vary a lot.-->

#### Origin and Evolution

> SARS-CoV-2 has a zoonotic origin. At least it is the most supported hypothesis by the collected genomic data.

Author: I am ok with that, it is more academic. Originally, I just wanted to emphasize the fact by making it a separate sentence.

After zoonotic transfer, SARS-CoV-2 continues evolving in the human population [@doi:10.1186/s12967-020-02344-6]. Luckily, the SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], and that restricts SARS-CoV-2 evolution pace. Nevertheless, genomic data show some significant statistical evidence of ongoing evolution. For example, the Spike G614 has already become dominant over D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043]. While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, there is no data yet to support or discard this hypothesis reliably [@doi:10.1016/j.cell.2020.06.040]. The study [@doi:10.1016/j.meegid.2020.104351] has reported more than 198 recurrent mutations using a dataset of 7666 curated sequences showing the dynamic of SARS-CoV-2 evolution. Overall, while it is obvious that SARS-CoV-2 exhibits moderate potential to further evolution, there is much uncertainty about SARS-CoV-2 evolution trends and consequences.

> The numerous SARS-CoV-2 mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351], observed in SARS-CoV-2 individual strains, can be used as signals during outbreak investigations and discovering of transmission patterns. The sequencing data can be found at [GISAID](https://www.gisaid.org/), [NCBI](https://www.ncbi.nlm.nih.gov/sars-cov-2/), and [COVID-19 data portal](https://www.covid19dataportal.org/). The databases contain more than 75,000 strains sampled and sequenced from the patients from December 2019 to July 2020. This data helped determine the source of local COVID-19 outbreaks in Connecticut, [@doi:10.1016/j.cell.2020.04.021], New Your City area [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100]. The tracking of SARS-CoV-2 mutations is recognized as an important tool for controlling outbreaks that is supposed to give important insides into the paths of SARS-CoV-2 spread [@doi:10.1016/S2666-5247(20)30054-9].

Reviewer: ```suggestion
The numerous SARS-CoV-2 mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351], observed in SARS-CoV-2 individual strains, can be used as signals during outbreak investigations and discovering of transmission patterns. The sequencing data can be found at [GISAID](https://www.gisaid.org/), [NCBI](https://www.ncbi.nlm.nih.gov/sars-cov-2/), and [COVID-19 data portal](https://www.covid19dataportal.org/). The databases contain more than 75,000 strains sampled and sequenced from the patients from December 2019 to July 2020. This data helped determine the source of local COVID-19 outbreaks in Connecticut, [@doi:10.1016/j.cell.2020.04.021], New Your City area [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100]. The tracking of SARS-CoV-2 mutations is recognized as an important tool for controlling outbreaks that is supposed to give important insides into the paths of SARS-CoV-2 spread [@doi:10/d5mg].
```
The parentheses in this DOI cause problems with Pandoc's citations.  Switching to the short DOI format is our workaround.

Can you also please switch to one sentence per line to help the reviewers?

The possibility of asymptomatic cases is also being investigated.
The transmission and mortality rate estimations of COVID-19 remain to be determined.
<!-- I think there are better estimates now (about 50% asymptomatic I think, once you take out the presymptomatic).-->
<!-- Most credible IFR estimates I've seen are between 0.5-1.5%, depending on the population. CFRs will be greater and vary a lot.-->

#### Origin and Evolution
SARS-CoV-2 has a zoonotic origin. At least it is the most supported hypothesis by the collected genomic data.
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].

> Human SARS-CoV-2 is genetically closer to bat RATG13 than to any other known virus, including human SARS-CoV. The similarity between SARS-CoV-2 and RATG13 is as high as 96.2%, while the similarity between SARS-CoV-2 and SARS-CoV is only 79% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].

Author: Thanks! Now I get the idea of breaking the paragraph into lines. Sorry, did not understand it before.

Nevertheless, some fragments between SARS-CoV-2 and RATG13 have a difference of up to 17%, suggesting a complex natural selection process during zoonotic transfer. 
While the S fragment is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about S origin in SARS-CoV-2.
Potentially it can be a result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].

> Anyway, while there is substantial support of the zoonotic origin of SARS-CoV-2, there is a lot of unknown, causing issues regarding the SARS-CoV-2 evolutionary pattern [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7].

Author: Agree, I will clarify this sentence and insert some citations from [@doi:10.1093/nsr/nwaa036].

After zoonotic transfer, SARS-CoV-2 continues evolving in the human population [@doi:10.1186/s12967-020-02344-6].

> Luckily, the SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], and that restricts SARS-CoV-2 evolution pace. 

Author: Thanks!

Nevertheless, genomic data show some significant statistical evidence of ongoing evolution.

> For example, the Spike G614 has already become dominant over D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].

Author: Sure I will edit the text.

While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, there is no data yet to support or discard this hypothesis reliably [@doi:10.1016/j.cell.2020.06.040].
The study [@doi:10.1016/j.meegid.2020.104351] has reported more than 198 recurrent mutations using a dataset of 7666 curated sequences showing the dynamic of SARS-CoV-2 evolution.

> Overall, while it is obvious that SARS-CoV-2 exhibits moderate potential to further evolution, there is much uncertainty about SARS-CoV-2 evolution trends and consequences.

Reviewer 2: ```suggestion
Overall, while it is obvious that SARS-CoV-2 exhibits moderate potential for further evolution, much uncertainty remains about SARS-CoV-2's evolutionary trends and their consequences.
```

> The numerous SARS-CoV-2 mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351], observed in SARS-CoV-2 individual strains, can be used as signals during outbreak investigations and discovering of transmission patterns.

Reviewer 2: ```suggestion
The numerous mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351] observed in SARS-CoV-2 individual strains can be used as signals during outbreak investigations and discovering of transmission patterns.
```
Just to check -- I kept seeing people argue about whether there were separate strains -- did the scientific consensus come down one way or the other about this? If not, we might want to add some sort of qualifier!

Author: As far as I understand the terminology. The viral strain is just a sequence with mutations, while the viral isolate is an instance with diverged biological behavior. But I agree that it may confuse people and require defined terminology. I will try to avoid it.

The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].
The databases contain more than 75,000 strains sampled and sequenced from the patients from December 2019 to July 2020.

> This data helped determine the source of local COVID-19 outbreaks in Connecticut, [@doi:10.1016/j.cell.2020.04.021], New Your City area [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].

Reviewer 2: ```suggestion
This data helped determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New Your City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
```
Proposal to be slightly less US centric, although we are unfortunately pretty US centric throughout the manuscript!

Author: Thanks!

> The tracking of SARS-CoV-2 mutations is recognized as an important tool for controlling outbreaks that is supposed to give important insides into the paths of SARS-CoV-2 spread [@doi:10/d5mg].

Reviewer 2: ```suggestion
The tracking of SARS-CoV-2 mutations is recognized as an important tool for controlling outbreaks that may provide valuable insights into the paths of SARS-CoV-2's spread [@doi:10/d5mg].
```
